Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabStrip.css

```diff
@@ -28,7 +28,7 @@
   display: flex;
   justify-content: center;
   align-items: center;
-  height: calc(var(--remotion-themed-line-height) + var(--remotion-themed-spacing));
+  min-height: calc(var(--remotion-themed-line-height) + var(--remotion-themed-spacing));
```

Comment:
tabstrip-items: when can this line-wrap?

Reply:
It wraps when there is not enough space (parent container is limited in size), but it had a rendering problem where the `height` was fixed, which caused the text to overflow from the element. Thus, this change.

Reply:
hmm... we have another spot where the text escapes the border in such a case. we should leave this for now and please make a note in our common.css task with the open todos that I need to check how we deal with overflow-text for such elemnts. it may well be that the text will be cut off isntead of allowed to wrap in such cases.